source/solvers/fluid_dynamics_matrix_free.cc

```diff
@@ -444,6 +444,24 @@ convert_vector_dealii_to_trilinos(
   out.import_elements(rwv, VectorOperation::insert);
 }
 
+/**
+ * @brief Helper function that allows to convert Trilinos vectors to deal.II vectors.
+ *
+ * @tparam number Abstract type for number across the class (i.e., double).
+ * @param out Destination LinearAlgebra::distributed::Vector<number> vector.
+ * @param in Source TrilinosWrappers::MPI::Vector vector.
+ */
+template <typename number>
+void
+convert_vector_trilinos_to_dealii(
+  LinearAlgebra::distributed::Vector<number> &out,
+  const TrilinosWrappers::MPI::Vector        &in)
```

Comment:
I guess mentally I would be tempted to put the out as the last one and to put the "const" argument first. This is generally what we tend to do, but I am not sure if this is really necessary :)

Reply:
I don't think this changes anything... I just want to be consistent with the other `convert_vector_dealii_to_trilinos(..)` function. But if you really prefer it the other way around then I will change it for both

Reply:
Nah it's fair. No big deal her. We never had a convention for this anyway...